DFC.ServiceTaxonomy.ApiFunction/CypherQueries/GetAllOccupations.json

```diff
@@ -0,0 +1,3 @@
+﻿{ 
+   "query":"MATCH (o:esco__Occupation) WITH{ occupation:collect({ uri:o.uri, occupation:head(o.skos__prefLabel), alternativeLabels:o.skos__altLabel, lastModified:head(o.dct__modified) }) } as occupations return occupations"
```

Comment:
This query returns json with the occupations array named 'occupation', rather than 'occupations'.
(The OpenAPI spec uses the correct name 'occupations' in the response schema.)

Reply:
The latest commit has changed the occupation label to 'occupations', rather than the occupation array, so now it returns...
```
{
  "occupation": [
    { ...
      "occupations": "pharmaceutical quality specialist",
```
whereas is needs to return
```
{
  "occupations": [
    { ...
      "occupation": "pharmaceutical quality specialist",

```